addon/components/lmb-plugin/search-modal.ts

```diff
@@ -39,13 +39,24 @@ export default class LmbPluginSearchModalComponent extends Component<Args> {
   }
 
   fetchData = restartableTask(async () => {
-    return fetchMandatees({ endpoint: this.args.config.endpoint });
+    try {
+      return fetchMandatees({ endpoint: this.args.config.endpoint });
+    } catch (err) {
+      console.error('got an error fetching mandatees', err);
+      this.error = err;
+      throw err;
+    }
   });
 
   search = restartableTask(async () => {
     // Can't do what I want, so if the user modifies the filter before resolving the query will run again
     if (!this.fetchData.lastComplete) {
-      await this.fetchData.perform();
+      try {
+        await this.fetchData.perform();
+      } catch (err) {
+        console.error('got an error fetching data', err);
+        this.error = err;
```

Comment:
This doesn't seem necessary to me? As we already are setting the `error` field in the `fetchData` task?

Reply:
Actually, weirdly, the other catch doesn't run. There must be something weird about how TS or ember is transpiling the code as if I `await` the `fetchMandatees()` call into a variable and then return it, the other catch runs as expected. I threw this in as a quick fix, I'll make a more carefully thought out version.

Reply:
Huh ok!